api/swagger-spec/storage.k8s.io_v1beta1.json

```diff
@@ -675,7 +675,7 @@
      },
      "metadata": {
       "$ref": "v1.ObjectMeta",
-      "description": "Standard object's metadata. More info: http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#metadata"
```

Comment:
@nikhiljindal the script you mention is probably https://github.com/kubernetes/kubernetes.github.io/blob/master/update-imported-docs.sh although I don't see a command replacing that pattern. For this PR I basically figured this link is no longer valid and just pointed it to the https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md#metadata instead.